config/initializers/truemail.rb

```diff
@@ -0,0 +1,77 @@
+require 'truemail'
+
+Truemail.configure do |config|
+  # Required parameter. Must be an existing email on behalf of which verification will be performed
+  config.verifier_email = 'info@internet.ee'
```

Comment:
Maybe re-using action_mailer_default_from what is defined in application.yml would be good idea ?
Not sure what they mean by existing email, if the account has to be present i think we maybe need to create mailbox for no-reply then, at least i would not mix role mail address and "system" mail address.

Reply:
Yes, suppose that's a good idea.